src/components/CardGrid/CardGrid.test.tsx

```diff
@@ -13,11 +13,11 @@ describe('<CardGrid>', () => {
       <CardGrid
         playlist={playlist}
         onCardHover={vi.fn()}
-        onCardClick={vi.fn()}
         isLoading={false}
         accessModel={'SVOD'}
         isLoggedIn={true}
         hasSubscription={true}
+        getUrl={(item: PlaylistItem) => `https://test.dummy.jwplayer.com?media_id=${item.id}`}
```

Comment:
Sorry, this should have been `mediaid`, not just `id`. Did you notice all the undefined values in the snapshot?
```suggestion
        getUrl={(item: PlaylistItem) => `https://test.dummy.jwplayer.com?media_id=${item.mediaid}`}
```